components/util/thread.rs

```diff
@@ -5,16 +5,41 @@
 use ipc_channel::ipc::IpcSender;
 use serde::Serialize;
 use std::borrow::ToOwned;
+use std::io::{Write, stderr};
+use std::panic::{PanicInfo, take_handler, set_handler};
 use std::sync::mpsc::Sender;
 use std::thread;
 use std::thread::Builder;
 use thread_state;
-
+#[allow(unused_must_use)]
```

Comment:
I skip the result of `write!`. I don't want to panic within a panic handler, and if stderr is broken I don't see much point of trying to output an error either.